test/extended/authorization/rolebinding_restrictions.go

```diff
@@ -0,0 +1,282 @@
+package authorization
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+
+	rbacv1 "k8s.io/api/rbac/v1"
+	kerrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/fields"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/watch"
+	"k8s.io/client-go/tools/cache"
+	watchtools "k8s.io/client-go/tools/watch"
+	kapi "k8s.io/kubernetes/pkg/apis/core"
+
+	authorizationapi "github.com/openshift/origin/pkg/authorization/apis/authorization"
+	exutil "github.com/openshift/origin/test/extended/util"
+)
+
+var _ = g.Describe("[Feature: RoleBinding Restrictions] RoleBindingRestrictions should be functional", func() {
+	defer g.GinkgoRecover()
+	var (
+		oc      = exutil.NewCLI("rolebinding-restrictions", exutil.KubeConfigPath())
+		cleanup = func(ns string) {
```

Comment:
_probably_  but I've since removed this cleanup, since I was cleaning up users created by test framework that I did not create, and after asking around in slack, determined that i should not be cleaning those up (or, the cleanup needs to happen elsewhere)  you'll notice if you run e2es locally after if you `oc get users` you'll see a slew of (non-idp)users/uids leftover.